client/src/editor/components/ChoiceBuilder/ChoiceBuilder.module.css

```diff
@@ -4,7 +4,7 @@
 
 .existingChoicesList {
   list-style: none;
-  background-color: #e2e2e2;
+  background-color: var(--color-light-feature-background);
```

Comment:
CSS Modules already has supoort for importing and exporting values. It's a transparent API so you don't have to use `var(--`. You would basically do something like.
```
@value colorLightFeatureBackgroundd from '../shared/components/colors';
```
The nice part about this is you can actually import from separate node modules if needed. This is useful if we ever want to have the components as a separate library.

Reply:
well that is friggin good to know